src/jit/codegencommon.cpp

```diff
@@ -7418,7 +7418,7 @@ void CodeGen::genProfilingEnterCallback(regNumber initReg, bool* pInitRegZeroed)
         return;
     }
 
-#if defined(_TARGET_AMD64_) && !defined(UNIX_AMD64_ABI) // No profiling for System V systems yet.
```

Comment:
Since this code doesn't really depend on FEATURE_SIMD, it would seem to make sense to change the setting of `storeIns` below to something like:
```
instruction store_ins = ins_Store(storeType);
#ifdef FEATURE_SIMD
if ((storeType == TYP_SIMD8) && genIsValidIntReg(argReg))
{
    storeType = INS_mov;
}
#endif
```
and similarly for `loadIns`